x-pack/plugins/security_solution/public/common/components/import_data_modal/index.tsx

```diff
@@ -91,6 +91,7 @@ export const ImportDataModalComponent = ({
     setOverwriteExceptions(false);
     setOverwriteActionConnectors(false);
     setActionConnectorsWarnings([]);
+    setIsImporting(false);
```

Comment:
Do you think adding a test for this particular scenario same makes sense so that this does not happen again?

Reply:
I looked into that but the loading state is only encoded in CSS class names that can change in the future 😞  (they contain hashes)